packages/fxa-graphql-api/src/auth/session-token.strategy.ts

```diff
@@ -13,11 +13,13 @@ export interface SessionTokenResult {
   session: SessionToken;
 }
 
+export const SESSION_TOKEN_REGEX = /^(?:[a-fA-F0-9]{2})+$/;
```

Comment:
I think I'm missing something here&hellip; So doing `(...{2})+` ensures that its an even number string length?

nit: Also, not sure it's worth doing a case insensitive RegExp to avoid needing to specify `a-f` and `A-F` separately.

```js
/^(?:[a-f0-9]{2})+$/i;
```

Reply:
This regex has in place for quite sometime, and I'd be **very** hesitant to change it. I simply moved it up in the file so it could be reused...